apidocs.yaml

```diff
@@ -0,0 +1,826 @@
+      responses:
+        "200":
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/RetrievedApplication"
+      security:
+        - auth:
+          - read:applications
+  /api/v1/applications/{id}/save-forms-and-duos:
+    post:
+      summary: save application forms and duos
+      operationId: save_application_forms_and_duos_v1
+      tags:
+        - "forms"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to update
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/SaveFormsAndDuos"
+      responses:
+        "204":
+          description: Successful Response (no content)
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/submit:
+    post:
+      summary: Submit application
+      operationId: submit_application_v1
+      tags:
+        - "actions"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to submit
+          required: true
+          schema:
+            type: string
+      responses:
+        "204":
+          description: Successful Response (no content)
+        "422":
+          description: Validation warnings
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ValidationWarnings"
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/create-application:
+    post:
+      summary: Create application
+      operationId: create_application_v1
+      tags:
+        - "actions"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateApplication"
+      responses:
+        "204":
+          description: Successful Response (no content)
+        "422":
+          description: Validation warnings
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ValidationWarnings"
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/cancel:
+    post:
+      summary: Cancel application
+      operationId: cancel_application_v1
+      tags:
+        - "actions"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to submit
+          required: true
+          schema:
+            type: string
+      responses:
+        "204":
+          description: Successful Response (no content)
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/copy-as-new:
```

Comment:
is it duplicate or something else?

Reply:
Nope, this is an action or a command to copy an existing application as a new one. Do you think this endpoint name is not appropriate?

Reply:
no its fine